lib/ice_cube.rb

```diff
@@ -68,7 +71,7 @@ module Validations
   # Defines the format used by IceCube when printing out Schedule#to_s.
   # Defaults to '%B %e, %Y'
   def self.to_s_time_format
-    @to_s_time_format ||= '%B %e, %Y'
+    I18n.t("ice_cube.date.formats.default")
```

Comment:
Yeah, actually caching is a really bad idea if you're using multiple locales in the same process (e.g. tests) - the second locale's date format gets ignored. My bad.